locales/nb-no.json

```diff
@@ -0,0 +1,797 @@
+    },
+    "page-no-duplicate-main": {
+      "pass": "Dokumentet har ikke mer enn ett 'main'-landmark",
+      "fail": "Dokumentet har mer enn ett 'main'-landmark"
+    },
+    "tabindex": {
+      "pass": "Elementet har ikke et 'tabindex' større end 0",
+      "fail": "Elementet har et 'tabindex' større end 0"
+    },
+    "alt-space-value": {
+      "pass": "Elementet har en alt-attributt med gyldig verdi",
+      "fail": "Elementet har en alt-attributt som kun inneholder et mellomrom, som ikke ignoreres av alle skjermlesere"
+    },
+    "duplicate-img-label": {
+      "pass": "Elementet duplikerer ikke den eksisterende tekst fra <img> elementets alt-tekst",
+      "fail": "Elementet inneholder et <img> element med alt-tekst, som duplikerer den eksisterende tekst"
+    },
+    "explicit-label": {
+      "pass": "Form-elementet har en eksplisitt <label>",
+      "fail": "Form-elementet har ikke en eksplisitt <label>"
+    },
+    "help-same-as-label": {
+      "pass": "Hjelpeteksten ('title' eller 'aria-describedby') duplikerer ikke label-teksten",
+      "fail": "Hjelpeteksten ('title' eller 'aria-describedby') er den samme som label-teksten"
+    },
+    "hidden-explicit-label": {
+      "pass": "Form-elementet har en synlig og eksplisitt <label>",
+      "fail": "Form-elementet har en eksplisitt <label>, som er skjult"
+    },
+    "implicit-label": {
+      "pass": "Form-elementet har en implisitt (inne i en) <label>",
+      "fail": "Form-elementet har ikke en implisitt (inne i en) <label>"
+    },
+    "label-content-name-mismatch": {
+      "pass": "Elementet inneholder synlig tekst som del av dets tilgjengelige navn",
+      "fail": "Tekst i elementet er ikke inkluderet i det tilgjengelige navn"
+    },
+    "multiple-label": {
+      "pass": "Form-feltet har ikke flere label-elementer",
+      "incomplete": "Flere label-elementer er ikke bredt understøttet i tilgjengelighetsteknologier. Sørg for at det første label-element inneholder all nødvendige informasjon."
+    },
+    "title-only": {
+      "pass": "Form-elementet bruker ikke utelukkende 'title'-attributten som label",
+      "fail": "Kun 'title'-attributten er brukt som label for form-elementet"
+    },
+    "landmark-is-unique": {
+      "pass": "Landmarks skal ha en unik rolle eller 'role'/'label'/'title'-attribut-kombinasjon (som tilgjengelig navn)",
+      "fail": "Et landmark skal ha en unik 'aria-label', 'aria-labelledby', eller 'title' for at gjøre landmarks adskillelige"
+    },
+    "has-lang": {
+      "pass": "<html> elementet har en 'lang'-attributt",
+      "fail": "<html> elementet har ikke en 'lang'-attributt"
+    },
+    "valid-lang": {
+      "pass": "'lang'-attributtens verdi er inkluderet i listen over godkjente språk",
+      "fail": "'lang'-attributtens verdi er ikke inkluderet i listen over godkjente språk"
+    },
+    "xml-lang-mismatch": {
+      "pass": "Attributterne 'lang' og 'xml:lang' har samme basisspråk",
```

Comment:
I suggest replacing "basespråk" with "språk / dialekt" as mentioned in https://dequeuniversity.com/rules/axe/4.0/html-xml-lang-mismatch